css-positioning/css-positioning_absolute-no-advanced-requirements/css/styles.css

```diff
@@ -41,24 +41,40 @@ main {
 
 .box--top-center {
   background-color: #85c1e9;
+  position: relative;
```

Comment:
Yes , I forgot to say, the position for all blocks should be ‘absolute’. As I understood from the title of the challenge.